test/read_and_write/test_pcs_converter.py

```diff
@@ -398,6 +398,40 @@ def test_build_new_GreaterThanIntCondition(self):
         value = pcs_new.write(cs)
         self.assertEqual(expected, value)
 
+    def test_read_new_configuration_space_forbidden(self):
+        cs_with_forbidden = ConfigurationSpace()
+        int_hp = UniformIntegerHyperparameter('int_hp', 0, 50, 30)
+        float_hp = UniformFloatHyperparameter('float_hp', 0., 50., 30.)
+        cat_hp_str = CategoricalHyperparameter('cat_hp_str', ['a', 'b', 'c'], 'b')
+        ord_hp_str = OrdinalHyperparameter('ord_hp_str', ['a', 'b', 'c'], 'b')
+
+        cs_with_forbidden.add_hyperparameters([int_hp, float_hp, cat_hp_str, ord_hp_str])
+
+        int_hp_forbidden = ForbiddenAndConjunction(ForbiddenEqualsClause(int_hp, 1))
```

Comment:
Not sure if this really needs to be wrapped inside a `ForbiddenAndConjunction`.

Reply:
Otherwise, the two generated ConfigSpace does not equal. the pcs_reader generates the forbidden_clauses with the following code:
https://github.com/dengdifan/ConfigSpace/blob/pcs_read/ConfigSpace/read_and_write/pcs_new.py#L450

Reply:
Thanks for the explanation.